doc/languages-frameworks/go.xml

```diff
@@ -66,6 +66,14 @@ pet = buildGoModule rec {
     </callout>
    </calloutlist>
   </para>
+
+  <para>
+    <varname>modSha256</varname> can also take <varname>null</varname> as an input.
+
+    When `null` is used as a value, the derivation won't be a
+    fixed-output derivation but disable the build sandbox instead. Don't use
```

Comment:
```suggestion
    fixed-output derivation but disable the build sandbox instead. This can be useful outside
    of nixpkgs where re-generating the modSha256 on each mod.sum changes is cumbersome,
    but will fail to build by Hydra, as builds with a disabled sanboxe are discouraged.
```